cpp/CMakeLists.txt

```diff
@@ -152,6 +163,13 @@ include(cmake/modules/ConfigureCUDA.cmake)
 # add third party dependencies using CPM
 rapids_cpm_init()
 
+# Not using rapids-cmake since we never want to find, always download.
+CPMAddPackage(
+  NAME rapids_logger GITHUB_REPOSITORY vyasr/rapids-logger GIT_SHALLOW FALSE GIT_TAG feat/cuml
+  VERSION feat/cuml
```

Comment:
Why feat/cuml here?

Reply:
This is only because I started work on https://github.com/rapidsai/cuml/pull/6162 before this PR, so that's the branch name for https://github.com/rapidsai/rapids-logger/pull/1. I'll merge that PR and update the commit hash here before this PR is merged.

Reply:
Done.